crates/compiler/can/tests/snapshots/test_suffixed__suffixed_tests__apply_argument_multiple.snap

```diff
@@ -30,13 +30,13 @@ Defs {
                     ident: "await",
                 },
                 [
-                    @17-18 Var {
+                    @17-19 Var {
```

Comment:
Huh, so it seems like this is in Task mode, drops the !, but the region includes the !, which is why this region is 2 chars instead of 1. It looks like this is necessary for compatibility, but yeah, it'll be nice to get rid of the old suffixing code.

Reply:
Yeah, this is because the parser always consumes the `!` as part of the identifier, which is how it works in purity inference mode. If the platform is `Task`-based, canonicalization then desugars the `!`-suffix identifiers into the actual `TrySuffix` expressions, so that existing code doesn't break while we transition.

Reply:
Maybe we should update the `Region` to reflect this too.

Reply:
Done in https://github.com/roc-lang/roc/pull/7170/commits/ed8f8d6c9414d81c9a4cec2500a156cdba89dfa9